src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -201,6 +202,16 @@ private void logError(SyaCaseWrapper appeal, IdamTokens idamTokens) {
         List<SscsCaseData> caseDetailsList = citizenCcdService.findCase(idamTokens);
 
         log.info("GET all Draft cases with IDAM Id {} and roles {}", idamTokens.getUserId(), idamTokens.getRoles());
+        StringBuffer logStr = new StringBuffer("");
+        caseDetailsList.stream().filter(c -> c.getAppeal() == null
```

Comment:
Why we are filtering out the caseDetailsList, the caseDetailsList should not be filter out for logs